PS4k/resource/bundle/LangBundle.properties

```diff
@@ -115,31 +117,31 @@ selection_tool_rotatez_tooltip = Rotate Around Z-Axe
 selection_tool_mirrorx_tooltip = Mirror X
 selection_tool_mirrory_tooltip = Mirror Y
 selection_tool_mirrorz_tooltip = Mirror Z
-selection_tool_mirrorx90_tooltip = Rotate Around X-Axe By 90�
-selection_tool_mirrorx180_tooltip = Rotate Around X-Axe By 180�
-selection_tool_mirrorx270_tooltip = Rotate Around X-Axe By 270�
-selection_tool_mirrory90_tooltip = Rotate Around Y-Axe By 90�
-selection_tool_mirrory180_tooltip = Rotate Around Y-Axe By 180�
-selection_tool_mirrory270_tooltip = Rotate Around Y-Axe By 270�
-selection_tool_mirrorz90_tooltip = Rotate Around Z-Axe By 90�
-selection_tool_mirrorz180_tooltip = Rotate Around Z-Axe By 180�
-selection_tool_mirrorz270_tooltip = Rotate Around Z-Axe By 270�
+selection_tool_mirrorx90_tooltip = Rotate Around X-Axe By 90�
+selection_tool_mirrorx180_tooltip = Rotate Around X-Axe By 180�
+selection_tool_mirrorx270_tooltip = Rotate Around X-Axe By 270�
+selection_tool_mirrory90_tooltip = Rotate Around Y-Axe By 90�
+selection_tool_mirrory180_tooltip = Rotate Around Y-Axe By 180�
+selection_tool_mirrory270_tooltip = Rotate Around Y-Axe By 270�
+selection_tool_mirrorz90_tooltip = Rotate Around Z-Axe By 90�
+selection_tool_mirrorz180_tooltip = Rotate Around Z-Axe By 180�
+selection_tool_mirrorz270_tooltip = Rotate Around Z-Axe By 270�
 
 selection_tool_rotatex_caption = Rotate X
 selection_tool_rotatey_caption = Rotate Y
 selection_tool_rotatez_caption = Rotate Z
 selection_tool_mirrorx_caption = Mirror X
 selection_tool_mirrory_caption = Mirror Y
 selection_tool_mirrorz_caption = Mirror Z
-selection_tool_mirrorx90_caption = Rotate X (90�)
-selection_tool_mirrorx180_caption = Rotate X (180�)
-selection_tool_mirrorx270_caption = Rotate X (270�)
-selection_tool_mirrory90_caption = Rotate Y (90�)
-selection_tool_mirrory180_caption = Rotate Y (180�)
-selection_tool_mirrory270_caption = Rotate Y (270�)
-selection_tool_mirrorz90_caption = Rotate Z (90�)
-selection_tool_mirrorz180_caption = Rotate Z (180�)
-selection_tool_mirrorz270_caption = Rotate Z (270�)
+selection_tool_mirrorx90_caption = Rotate X (90�)
+selection_tool_mirrorx180_caption = Rotate X (180�)
+selection_tool_mirrorx270_caption = Rotate X (270�)
+selection_tool_mirrory90_caption = Rotate Y (90�)
+selection_tool_mirrory180_caption = Rotate Y (180�)
+selection_tool_mirrory270_caption = Rotate Y (270�)
+selection_tool_mirrorz90_caption = Rotate Z (90�)
+selection_tool_mirrorz180_caption = Rotate Z (180�)
+selection_tool_mirrorz270_caption = Rotate Z (270�)
```

Comment:
What's up with these. Why are they different? Is this just a Mac encoding thing?